### PR TITLE
Added links to WASM SDK.

### PR DESCRIPTION
**Pull request by icarusz**
Added links to WASM SDK from API page, the front page and updated color scheme to match website.

### CODE REVIEW

**Comment by icarusz** [comment]
Screen Shot 2019-10-10 at 5 36 12 PM
This is the new front page.  Titles and stuff a different color to be closer to website.
This down plays the hello world tutorial in the first section (but there is still a link) and ends with a call out to the WASM video.  Its just a start but better something up to get started with.

File: api_reference.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ next few months.
 
 ***
 
+### Latest Release
+### WASM SDK - Tupelo DLT right from the browser
+
+#### [SOURCE](https://github.com/QuorumControl/tupelo-wasm-sdk)
+#### [GETTING STARTED GUIDE](https://www.quorumcontrol.com/blog/2019/8/22/new-wasm-based-sdk-for-tupelo)
```
Comment (tobowers) [inline api_reference.md:24]: tupelo.org ?

File: index.md [hunk 2]
```diff
@@ -10,16 +10,16 @@ permalink:
...
-Start building now by adding a “Hello World” message and get it signed in seconds.
+{: .fs-5 .fw-300 }
+
+Start building now by adding a [“Hello World” message](tutorials) and get it signed in seconds.
 {: .fs-5 .fw-300 }
 Need more information first?  
 Read below to see how we’re different or get access to important documentation.
 {: .fs-5 .fw-300 }
-**Your project will scale on the Tupelo platform.**  Keep us honest and try it out.
+Want to jump right in and build?  Get started using your preferred web development framework and access
```
Comment (tobowers) [inline index.md]:
```suggestion
Want to jump right in and build?  Start accessing
```

**Review by tobowers — APPROVED** [review]
step in the right direction.

**Pull request merged**